### PR TITLE
Remove Google+ API

### LINKED ISSUE #10079: Impact of Google+ API deprecation on this flow?
With the Google+ APIs set to shut down on March 7, 2019, I'm unclear on what changes are needed for this .NET Core flow to continue authenticating Google users. There are several Google dev docs/samples but I'm hoping to avoid a separate flow for Google versus Facebook and Microsoft Account. Any guidance on where to start?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6d19b295-f935-3755-4ca4-9d0717b21e75
* Version Independent ID: 747a179e-90b7-de07-e270-82414963048a
* Content: [Google external login setup in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-2.2)
* Content Source: [aspnetcore/security/authentication/social/google-logins.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/security/authentication/social/google-logins.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #10079
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-1.0&branch=pr-en-us-10445)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@Tratcher 	the GitHub diffs are difficult to read, you're better off looking at the [Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-1.0&branch=pr-en-us-10445)

Don't miss:

`<!-- REVIEW: Should See the [GoogleOptions] be deleted? -->`
See the [GoogleOptions](/dotnet/api/microsoft.aspnetcore.builder.googleoptions) API reference for more information on configuration options supported by Google authentication. This can be used to request different information about the user.

**Comment by Rick-Anderson** [comment]
@01binary FYI

**Comment by Rick-Anderson** [comment]
@Tratcher applied your changes. Let me know if it's OK and I'll have the updates live 15 minutes later.

File: aspnetcore/security/authentication/social/google-logins.md [hunk 1]
```diff
@@ -4,156 +4,68 @@ author: rick-anderson
...
 
-[!INCLUDE[](includes/chain-auth-providers.md)]
-
-::: moniker-end
-
-::: moniker range="< aspnetcore-2.0"
+## Configure Google authentication
 
-The project template used in this tutorial ensures that [Microsoft.AspNetCore.Authentication.Google](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Google) package is installed.
+Add the Google service to `Startup.ConfigureServices`. See the code in [this GitHub issue](https://github.com/aspnet/AspNetCore/issues/6486).
```
Comment (Tratcher) [inline aspnetcore/security/authentication/social/google-logins.md]:
```suggestion
Add the Google service to `Startup.ConfigureServices`.
```
The link is redundant, you already have it at the top. At some point after everyone has updated we'll remove that top paragraph and be done with it.

**Pull request merged**